_includes/integrations/bootstrap-quick-start.md

```diff
@@ -0,0 +1,62 @@
+Using {{site.productname}} with bootstrap does not require any special configuration.
+
+The {{site.productname}} jQuery integration can be used with bootstrap. For information on using the jQuery integration, see: [jQuery integration]({{site.baseurl}}/integrations/jquery/)
+
+This procedure creates a basic Bootstrap integration containing a {{site.productname}} editor.
+
+## Setup
+
+1. In an HTML file, add a source script for Bootstrap JS.
+
+    If the project loads Bootstrap from <https://www.bootstrapcdn.com/>, use the script provided by the [Bootstrap CDN](https://www.bootstrapcdn.com/), which includes the `integrity` and `crossorigin` attributes.
+{% if productSource == "cloud" %}
+2. To source {{site.productname}} from the {{site.cloudname}}, add the following `script` element to the `<head>` of the document:
+
+    ```html
+    <script src="{{site.cdnurl}}" referrerpolicy="origin"></script>
+    ```
+
+    Replace `no-api-key` in the source script (`<script src=...`) with a {{site.cloudname}} API key, which is created when signing up to the [{{site.cloudname}}]({{site.accountsignup}}).
+
+    Signing up for a {{site.cloudname}} API key will also provide a trial of the [Premium Plugins]({{site.baseurl}}/plugins/premium/).
+{% else %}
+2. To source an independent deployment of {{site.productname}}, add the following source script to either the `<head>` or the end of the `<body>` of the HTML file, such as:
+
+    ```html
+    <script src="/path/to/tinymce.min.js"></script>
+    ```
+
+    For information on self-hosting {{site.productname}}, see: [Installing {{ site.productname }}]({{site.baseurl}}/general-configuration-guide/advanced-install/).
+{% endif %}
+3. Add an initialization point for {{site.productname}}, such as:
+
+    ```html
+    <div>
+      <textarea id="tiny"></textarea>
+    </div>
+    ```
+
+4. Add the {{site.productname}} init script.
+
+    ```html
+    <script>
+      tinymce.init({
+        selector: 'textarea#tiny'
+      });
+    </script>
+    ```
+
+## Using TinyMCE in a Bootstrap dialog
+
+To render {{site.productname}} instances inside Bootstrap UI dialogs, add the following code:
```

Comment:
This definitely isn't right... the whole Bootstrap 5 section is missing here: https://www.tiny.cloud/docs/integrations/bootstrap/#usingtinymceinabootstrapdialog

Reply:
fixed in https://github.com/tinymce/tinymce-docs/pull/2196/commits/9eac4d2b0b024228dc88cacbb7ef58c295ac5477